keras2onnx/ke2onnx/batch_norm.py

```diff
@@ -47,26 +47,23 @@ def convert_keras_batch_normalization(scope, operator, container):
     if not op.center:
         params.insert(1, np.zeros(params[1].shape, dtype=float))
 
-    gamma = params[0] / np.sqrt(params[3] + op.epsilon)
```

Comment:
According to the paper, https://arxiv.org/pdf/1502.03167v3.pdf, the formula in p3. the original calculation is correct in the inference mode. Do you have any example to show the new change would be more accurate?

Reply:
My understanding is as follows. The Keras op has the following parameters:

```python
param[0] := gamma
param[1] := beta
param[2] := moving_mean
param[3] := moving_variance
```

These are then applied as described in the paper (under `Algorithm 1`):
> x_norm = (x - moving_mean) / sqrt(moving_variance + epsilon)
> y = gamma * x_norm + beta

The ONNX operator has the same parameters, but with slightly different naming

```
scale := gamma
B     := beta
mean  := moving_mean
var   := moving_variance
```

However, the previous change appears to have interpreted these values instead as:

```python
scale = gamma / sqrt(moving_variance + epsilon) # Most similar to x_norm's definition?
B     = beta - gamma * moving_mean / sqrt(moving_variance + epsilon) # Unsure how to interpret this
# Perhaps trying to treat the scale and B(ias) tensors as the x_norm and y terms
# as opposed to as gamma and beta?
```

My understanding is it should just be a direct translation of gamma -> scale etc, but it's possible I missed the secret behind these formulas